docs/admin/federation-apiserver.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
-<strong>
-The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.3/docs/admin/federation-apiserver.md).
-
```

Comment:
@juanvallejo pro-tip:

Assuming you have your fork as the `origin` remote, and `upstream` points to this repo.

```
$ git fetch --all --prune
$ git rebase -i upstream/master
$ # make sure you just have your commits in the list
$ # resolve eventual conflicts
$ # re-run the update-generated-*.sh scripts
$ # review changes
$ git add .
$ git diff --cached
$ git commit --amend
$ git show
$ git push origin -f
```

Don't mess around with deleting and recreating branches, or you may end up losing access to this PR.